eZ/Publish/Core/Repository/SearchService.php

```diff
@@ -32,12 +33,6 @@
 class SearchService implements SearchServiceInterface
 {
     /**
-     * 2^30, since PHP_INT_MAX can cause overflows in DB systems, if PHP is run
-     * on 64 bit systems
-     */
-    const MAX_LIMIT = 1073741824;
```

Comment:
could have changed this to 1000 in retrospective if that would work with solr/es. Less changes and less breaks. It is anyway what legacy engine was setting it to. Only change would be that more the 100 would not be allowed.